### PR TITLE
Adding additional text to 'Keep alive' and 'Notifications'

### PR DESCRIPTION
**Pull request by peterroth**

### CODE REVIEW

File: app/src/ui-blokada/res/values/strings_notification.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <!-- Parameter: branded app name (Blokada). -->
 
     <string name="notification_keepalive_text">Keep alive</string>
-    <string name="notification_keepalive_description">Activate this switch if you experience problems with app staying active. The app will enable a permanent notification to prevent the system from killing it.</string>
+    <string name="notification_keepalive_description">Activate this switch if you experience problems with the app staying active in the background. Blokada will enable a permanent notification to prevent the system from killing it\; you can hide it by choosing Notifications above and there Settings.</string>
```
Comment (kar) [inline app/src/ui-blokada/res/values/strings_notification.xml]: no need for a backslash

File: app/src/ui-blokada/res/values/strings_notification.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <resources>
     <string name="notification_on_text">Notifications</string>
-    <string name="notification_on_description">When on, Blokada will display a notification every time something is blocked. Please note this is not the same as the VPN notification some devices show, and which cannot be turned off.</string>
+    <string name="notification_on_description">When on, Blokada will display a notification every time something is blocked. Please note this is not the same as the VPN notification some devices show, and which cannot be turned off.&lt;br&gt;You can configure which notifications should be displayed with the Settings below.</string>
```
Comment (kar) [inline app/src/ui-blokada/res/values/strings_notification.xml]: this: 
&lt;br&gt;
should be
&lt;br/&gt;

Comment (kar) [reply app/src/ui-blokada/res/values/strings_notification.xml]: Oh it encoded the characters in my comment. just add a slash before the second ampersand

Comment (peterroth) [reply app/src/ui-blokada/res/values/strings_notification.xml]: Fixed with 41a53ae20add8b5b199bdfd52b9a8b85edf2e5b5.

File: app/src/ui-blokada/res/values/strings_notification.xml (hunk 1 above)
Comment (peterroth) [reply app/src/ui-blokada/res/values/strings_notification.xml]: Fixed with 41a53ae20add8b5b199bdfd52b9a8b85edf2e5b5.

**Pull request merged**